recipes/gromacswrapper/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "gromacswrapper" %}
+{% set version = "0.8.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/GromacsWrapper-{{ version }}.tar.gz
+  sha256: 4d695db79e536ee9752a82551de5f58451bd1c0aa0b7d6cb30fe7e3b9e19768f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - numkit
+    - numpy >=1.0
+    - python >=3.6
+    - six
+
+test:
+  imports:
+    - gromacs
+    - gromacs.fileformats
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Becksteinlab/GromacsWrapper
+  summary: A python wrapper around the Gromacs tools.
+  doc_url: https://pythonhosted.org/GromacsWrapper/
+  dev_url: https://github.com/Becksteinlab/GromacsWrapper
+  license: GPL-3.0-only
```

According to the source:
```suggestion
  license: GPL-3.0-or-later
```